package.json

```diff
@@ -2,16 +2,39 @@
   "name": "minecraft-data",
   "version": "2.70.0",
   "description": "Provide easy access to minecraft data in node.js",
-  "main": "index.js",
-  "tonicExampleFilename": "example.js",
+  "main": "build/main/index.js",
```

Comment:
Don’t think anything needs to change here other than running tsc and switching to standard for typescript